core/indexer/tests/contract_load_tests.rs

```diff
@@ -0,0 +1,83 @@
+//! Load tests for Kontor
+//!
+//! These tests are designed to stress-test the system.
+//! Run with `--release` for meaningful performance data.
+//!
+//! The CI configuration ensures these always run optimized.
+
+use testlib::*;
+use tracing::info;
+
+interface!(name = "token", path = "../test-contracts/token/wit");
+
+/// Simple load test: process many blocks with many contract calls
+/// Each contract call creates a transaction and mines a block automatically in regtest
+#[testlib::test(contracts_dir = "test-contracts", mode = "regtest")]
+async fn test_token_contract_load() -> Result<()> {
+    logging::setup();
+
+    let num_operations = 500; // Total contract calls to execute
+
+    info!("Starting load test: {} contract operations", num_operations);
+
+    // Setup: create a token contract and some users
+    let minter = runtime.identity().await?;
+    let token = runtime.publish(&minter, "token").await?;
+
+    // Mint a large supply
+    let _ = token::mint(runtime, &token, &minter, "1000000000".into()).await?;
```

Comment:
For instances like this just do two `?` at the end like `.await??` and then you don't need the `let _ =`